dipy/viz/tests/test_actors.py

```diff
@@ -103,12 +103,18 @@ def test_slicer():
                                       value_range=(0., 1.))
     renderer.clear()
     slicer_lut = actor.slicer(data, lookup_colormap=lut)
-
     slicer_lut.display(10, None, None)
     slicer_lut.display(None, 10, None)
     slicer_lut.display(None, None, 10)
-
+    slicer_lut.opacity(0.5)
+    slicer_lut.tolerance(0.03)
     slicer_lut2 = slicer_lut.copy()
+    npt.assert_equal(slicer_lut2.GetOpacity(), 0.5)
+    npt.assert_equal(slicer_lut2.picker.GetTolerance(), 0.03)
+    slicer_lut.opacity(1)
+    slicer_lut.tolerance(0.025)
```

Comment:
Correct me if I am wrong, but aren't lines 114-117 redundant? @dmreagan 
116-117 can still kind of make sense, but the properties set in 114-115 aren't being used/asserted anywhere.

Reply:
Done